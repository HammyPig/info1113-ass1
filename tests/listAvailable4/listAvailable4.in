LIST AVAILABLE
ADD COLLECTION duplicates.csv
LIST AVAILABLE
ADD MEMBER coolman69
RENT 100000 111111
LIST AVAILABLE
RENT 100000 111112
LIST AVAILABLE
RELINQUISH ALL 100000
RENT 100000 111113
LIST AVAILABLE
RENT 100000 111111
RENT 100000 111112
LIST AVAILABLE