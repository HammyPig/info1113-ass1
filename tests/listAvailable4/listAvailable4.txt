deals with duplicate books of the same information