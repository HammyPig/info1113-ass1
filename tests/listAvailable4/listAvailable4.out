user: No books in system.

user: 3 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: Success.

user: The Iliad (Homer)

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: No books available.

user: Ending Library process.
